ion/ionops_test.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pytest
+
+import cirq
+import ion
+
+def test_CxNOTGate_init():
+    assert ion.CxNOTGate(exponent=8) == ion.CxNOTGate(exponent=0)
```

Comment:
This is testing equality, not initialization. Just check that the `.exponent` property is the value you passed in. Then add a `test_CxNotGate_eq` method using `eq = cirq.testing.EqualsTester()` and add a few equivalence groups to `eq`.